tests/foreman/cli/test_satellitesync.py

```diff
@@ -905,7 +905,7 @@ def test_positive_export_import_promoted_cv(
     @pytest.mark.tier3
     @pytest.mark.upgrade
     def test_positive_export_import_redhat_cv(
-        self, export_cleanup, config_export_import_settings, module_org, default_sat
+        self, export_cleanup, config_export_import_settings, function_org, default_sat
```

Comment:
Can't the org be reused in the whole module?

Reply:
Unfortunately, we need to use specific content for each test, therefore we need "clean" organization.